contrib/guix/libexec/prelude.bash

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
```

Comment:
In commit "guix: Add souce-able bash prelude and utils"

Nit: commit title contains a typo

Reply:
Fixed, thanks!